src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -155,44 +161,30 @@ private void resetToAbsolute() {
                 SwerveConstants.ANGLE_GEAR_RATIO);
         // Set the integrated angle encoder to the absolute position.
         CTREUtil.checkError(
-                () -> mAngleMotor.setSelectedSensorPosition((int) absolutePosition, 0,
+                () -> angleMotor.setSelectedSensorPosition((int) absolutePosition, 0,
                         50), 3);
     }
 
     /**
      * Configures the angle encoder to the given constants.
      */
     private void configAngleEncoder() {
-        CTREUtil.checkError(() -> angleEncoder.configFactoryDefault(30), 3);
         CTREUtil.checkError(
                 () -> angleEncoder.configFeedbackNotContinuous(true, 30), 3);
         CTREUtil.checkError(
                 () -> angleEncoder.configSelectedFeedbackSensor(
                         TalonSRXFeedbackDevice.CTRE_MagEncoder_Absolute, 0, 30),
                 3);
```

Comment:
should be part of motor config?

Reply:
a bit too specific no?

Reply:
I don't think that MotorConfig can be too specific